### PR TITLE
NoMultilineWhitespaceBeforeSemicolonsFixer - SpaceAfterSemicolonFixer - priority fix

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3129

### CODE REVIEW

File: src/Fixer/Semicolon/SpaceAfterSemicolonFixer.php [hunk 1]
```diff
@@ -44,6 +44,14 @@ public function getDefinition()
     /**
      * {@inheritdoc}
      */
+    public function getPriority()
+    {
+        return -1;
```
Comment (keradus) [inline src/Fixer/Semicolon/SpaceAfterSemicolonFixer.php]: please put comment with reasoning ("should run before/after XXX")

**Comment by keradus** [comment]
nice !

File: src/Fixer/Semicolon/SpaceAfterSemicolonFixer.php (hunk 1 above)
Comment (SpacePossum) [reply src/Fixer/Semicolon/SpaceAfterSemicolonFixer.php]: :+1: totally forgot that :)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**